quartz/src/main/java/org/quartz/simpl/SimpleVirtualThreadPool.java

```diff
@@ -0,0 +1,124 @@
+/* 
```

Comment:
I think a code format should be run on these so that the formatting is consistent.